test/metabase/api/dashboard_test.clj

```diff
@@ -2366,26 +2366,26 @@
   {:is_write true
    :parameters [{:id field-name :slug field-name :target ["variable" ["template-tag" field-name]] :type :text}]
    :dataset_query (mt/native-query
-                    {:query (format "insert into TYPES (%s) values ({{%s}})"
+                    {:query (format "insert into types (%s) values ({{%s}})"
                                     (str/upper-case field-name)
                                     field-name)
                      :template-tags {field-name {:id field-name :name field-name :type :text :display_name field-name}}})})
 
 (deftest dashcard-action-execution-type-test
   (mt/test-drivers (mt/normal-drivers-with-feature :actions)
-    (let [types [{:field-name "text" :base-type :type/Text ::good "hello"}
-                 {:field-name "json" :base-type :type/JSON ::good "{\"a\": 1}"}
-                 {:field-name "xml" :base-type :type/XML ::good "<xml></xml>"}
-                 {:field-name "boolean" :base-type :type/Boolean ::good true ::bad "not boolean"}
-                 {:field-name "integer" :base-type :type/Integer ::good 100}
-                 {:field-name "float" :base-type :type/Float ::good 0.4}
+    (let [types [{:field-name "atext" :base-type :type/Text ::good "hello"}
+                 {:field-name "ajson" :base-type :type/JSON ::good "{\"a\": 1}"}
+                 {:field-name "axml" :base-type :type/XML ::good "<xml></xml>"}
+                 {:field-name "aboolean" :base-type :type/Boolean ::good true ::bad "not boolean"}
+                 {:field-name "ainteger" :base-type :type/Integer ::good 100}
+                 {:field-name "afloat" :base-type :type/Float ::good 0.4}
```

Comment:
In this test we don't intend to check how the drivers can deal with reserved keywords.